src/components/pip/pip-parent.tsx

```diff
@@ -22,7 +22,12 @@ export class PipParent extends Component<PIPParentComponentProps> {
 
   componentDidMount() {
     const {player} = this.props;
+
+    const videoFilter = document.createElement('div');
```

Comment:
not a critical one, but since player applying the same styles we can move creation and appending element into ui.utils